mimic/rest/loadbalancer_api.py

```diff
@@ -54,10 +55,104 @@ def resource_for_region(self, region, uri_prefix, session_store):
         Get an :obj:`twisted.web.iweb.IResource` for the given URI prefix;
         implement :obj:`IAPIMock`.
         """
+        self.session_store = session_store
```

Comment:
Apologies, I messed this up before and got confused - I think with `_collections_from_tenant`, probably storing this on the `LoadBalancerApi` may not be necessary?  (so it doens't need to be stored here or in the `init`, and in `LoadBalancerControlApi.resource_for_region`, the `session_store` passed in the function parameter can be used instead of `self.lb_api.session_store` (since it should be the same session store either way).

Really really sorry I misunderstood this before.